server/src-lib/Hasura/GraphQL/Transport/WebSocket.hs

```diff
@@ -210,23 +211,36 @@ sendMsg :: (MonadIO m) => WSConn -> ServerMsg -> m ()
 sendMsg wsConn msg =
   liftIO $ WS.sendMsg wsConn $ WS.WSQueueResponse (encodeServerMsg msg) Nothing
 
+-- sendCloseWithMsg closes the websocket server with an error code that can be supplied as (Maybe Word16),
+-- if there is `Nothing`, the server will be closed with an error code derived from ServerErrorCode
 sendCloseWithMsg ::
   (MonadIO m) =>
   L.Logger L.Hasura ->
   WSConn ->
   ServerErrorCode ->
   Maybe ServerMsg ->
+  Maybe Word16 ->
```

Comment:
`Word16` is used in `sendCloseCode`, which is a part of the standard module (Network.WebSockets).

Reply:
Am I right to assume that this is a note?

Reply:
Yupp